grove/blackbox_algorithms/deutsch_jozsa.py

```diff
@@ -0,0 +1,69 @@
+"""Module for Deutsch-Jozsa Algorithm. Adapted from Harley Patton's deutsch_jozsa.py"""
+import pyquil.quil as pq
+from pyquil.gates import *
+import utils as bbu
+from blackbox import AbstractBlackBoxAlgorithm
+
+class DeutschJozsa(AbstractBlackBoxAlgorithm):
```

Comment:
@ncrubin I just talked with @joelin0 and we think we will keep these algorithms separate for now. I believe he is replacing this pull request soon. Let us know if you have any input on this.